src/pages/home/components/Sidebar/Sidebar.tsx

```diff
@@ -26,56 +23,44 @@ const HALF_SECOND = 500;
 const format = (userName: string): string => userName.toLowerCase().trim();
 
 function Sidebar(): React.ReactNode {
-  const { accounts, addAccount } = useSocialMediaStore();
+  const { accounts } = useSocialMediaStore();
   const [isOpen, setIsOpen] = useState(false);
   const [inputValue, setInputValue] = useState('');
-  const [filteredAccounts, setFilteredAccounts] = useState(accounts.data);
-  const inputSearchRef = useRef<TInputComponent | null>(null);
-  const isEmptyResult = isEmpty(filteredAccounts) && inputValue;
+
+  const filteredAccountsResult = useMemo(() => {
+    const data: {
+      socialMediaAccounts: StoreAccount[];
+      socialMediaId: string;
+    }[] = [];
+
+    for (const [socialMediaId, socialMediaAccounts] of Object.entries(
+      accounts.data
+    )) {
```

Comment:
Pq `for`? É pq do role do lint? Se é, só arranca essa regra

Reply:
Eu fiz com foreach mas quando eu enviei o codigo ele alterou para o for